### PR TITLE
[Notion] Update docs

### PR DESCRIPTION
**Pull request by 43-stuti**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: src/chapter-content/chapter_1_vectors.html [hunk 1]
```diff
@@ -4,7 +4,7 @@ <h1>Chapter 1. Vectors</h1>
...
+<p>This book is all about looking at the world around us and coming up with clever ways to simulate that world with code. Divided into three parts, the book will start by looking at basic physics—how an apple falls from a tree, a pendulum swings in the air, the earth revolves around the sun, etc. Absolutely everything contained within the first five chapters of this book requires the use of the most basic building block for programming motion—the <strong><em>vector</em></strong>. And so this is where I begin our story.</p><a data-type="indexterm" data-primary="Euclid"></a><a data-type="indexterm" data-primary="Euclidean vector"></a><a data-type="indexterm" data-primary="geometric vector"></a><a data-type="indexterm" data-primary="vectors" data-secondary="defined"></a>
```
Comment (shiffman) [inline src/chapter-content/chapter_1_vectors.html:7]: It works! I made some minor edits to Chapter 1 and they came in beautifully when running the GitHub action!

**Review by shiffman — APPROVED** [review]
@43-stuti @jasongao97 if you want to take a look! I tested the workflow by making some edits to the book and re-running!

File: src/chapter-content/chapter_1_vectors.html (hunk 1 above)
Comment (jasongao97) [reply src/chapter-content/chapter_1_vectors.html:7]: Looking great!

**Comment by shiffman** [comment]
This is an old one so closing!

**Pull request closed without merging**